### DISCUSSION TITLE
Netbox as external IPAM for vRealize Automation

### DISCUSSION THREAD

**Discussion by ciroiriarte** [netbox-community/netbox#6594] [Ideas]
Hello!,

I'm wondering if anybody has ever integrated Netbox as an IPAM provider for vRA 8.x.

VMware provides an external IPAM SDK for this purpose.

Ref: https://code.vmware.com/web/sdk/1.1.0/vmware-vrealize-automation-third-party-ipam-sdk

**Comment by rnhinson** [comment]
Hey, sorry to bump this old discussion, but I just finished writing an initial integration. Hoping to get some more features added over time and with community help! https://github.com/rnhinson/vra8-netbox

**Reply by mbriet** [reply]
hello,
I visit your github but I didn't understand how to build the package can you help me
